modules/ossm-extensions-migration-overview.adoc

```diff
@@ -0,0 +1,90 @@
+////
+This module included in the following assemblies:
+*service_mesh_/v2x/ossm-extensions.adoc
+////
+:_content-type: CONCEPT
+[id="ossm-extensions-migration-overview_{context}"]
+= Migrating from `ServiceMeshExtension` to `WasmPlugin` resources
+
+The `ServiceMeshExtension` API is deprecated as of {SMProductName} version 2.2 and will be removed in a future release. If you are using the `ServiceMeshExtention` API, you must migrate to the `WasmPlugin` API for your extensions.
```

Comment:
Does "for your extensions" mean "to continue using your WebAssembly extensions"? Consider adding the details for clarity.